lib/Crust/Middleware/AccessLog.pm6

```diff
@@ -38,21 +38,23 @@ my sub content-length(@res) {
 }
 
 method CALL-ME(%env) {
-    my $t0 = DateTime.now.Instant;
-    my @res = $.app()(%env);
-
-    # '%h %l %u %t "%r" %>s %b "%{Referer}i" "%{User-agent}i"'
-    my $logger = $.logger;
-    if !$logger.defined {
-        $logger = sub ($s) { %env<p6sgi.errors>.print($s) };
-    }
+    start {
+        my $t0 = DateTime.now.Instant;
+        my @res = $.app()(%env).result;
+
+        # '%h %l %u %t "%r" %>s %b "%{Referer}i" "%{User-agent}i"'
+        my $logger = $.logger;
+        if !$logger.defined {
+            $logger = sub ($s) { %env<p6sgi.errors>.print($s) };
```

Comment:
apps (/middlewares) should emit errors by `%env<p6sgi.errors>.emit($s)`. ref: #94